packages/evm/src/opcodes/codes.ts

```diff
@@ -291,6 +291,14 @@ const eipOpcodes: { eip: number; opcodes: OpcodeEntry }[] = [
       0xf7: { name: 'AUTHCALL', isAsync: true, dynamicGas: true },
     },
   },
+  {
+    eip: 4200,
+    opcodes: {
+      0x5c: { name: 'RJUMP', isAsync: false, dynamicGas: false },
+      0x5d: { name: 'RJUMPI', isAsync: false, dynamicGas: false },
+      0x5e: { name: 'RJUMPV', isAsync: false, dynamicGas: false },
```

Comment:
`RJUMPV` is "basically" an upgraded version of `RJUMPI`, the first byte after `RJUMPV` is whatever count the table size is, now you check the stack: is this stack item lower than the table size? In that case, lookup the `int16` of that table and then jump to it, otherwise, just jump right after the table. So, dynamic gas is not applicable here. (I had a lot of trouble understanding this opcode)

Reply:
Yeah, but isn't the *size* of this table dynamic and you could therefore bloat the state with a big table size? 🤔

Reply:
No, the size of this table is determined at code deployment. The `count` is thus hardcoded in the code itself. The `case` is an item which is popped from stack.

Note that the costs for these "big tables" is contained into deployment costs of the code: for each byte you deploy, you have to pay gas (I think it is something like 32 gas per byte or something)